test/e2e/virtual/apiexport/virtualworkspace_test.go

```diff
@@ -60,44 +62,50 @@ func TestAPIExportVirtualWorkspace(t *testing.T) {
 
 	cfg := server.DefaultConfig(t)
 
-	kcpClients, err := clientset.NewClusterForConfig(cfg)
+	// copy of original config to keep it for other uses
+	cfgCopy := rest.CopyConfig(cfg)
+
+	cfg.Wrap(func(rt http.RoundTripper) http.RoundTripper {
+		return kcp.NewClusterRoundTripper(rt)
+	})
```

Comment:
no aliases please. Make `DefaultConfig` do the right thing.

Reply:
What do you mean?

BTW, because discovery doesn't take in a context, we can't use the wrapper for it. That's why we keep cfgCopy around.